### ISSUE TITLE
Fatal Error caused by the plugin when creating menu items with "onOffice" via ACF

### ISSUE THREAD

**Issue by fredericalpers** [onOffice-Web-Org/oo-wp-plugin#684]
### Description
You get the following error when you open a submenu page in Wordpress if the main menu item contains the word "onOffice". However, the first submenu page that is opened when you click on the main menu item does not display an error. The 'menu_title' must not contain "onOffice".

### Error message:
`Fatal error: Uncaught Error: Call to undefined method acf_admin_options_page::doExtraEnqueues()
in /home/monikar/maklerwebsites/pt-orange/wp-content/plugins/onoffice-for-wp-websites/plugin/Controller/AdminViewController.php on line 344`

### Seemingly responsible line in the plugin:
https://github.com/onOffice-Web-Org/oo-wp-plugin/blob/5fddd37fb4cd3118d831a51f9f295f33f3b790da/plugin/Controller/AdminViewController.php#L329

### Create the menu items in the functions via ACF hooks:

```
function oo_acf_options_page()
{
    if (function_exists('acf_add_options_page')) {
        // onOffice WP-Website
        acf_add_options_page([
            'page_title' => __('onOffice WP-Website', 'oo_theme'),
            'menu_title' => __('onOffice WP-Website', 'oo_theme'),
            'menu_slug' => 'oo-website',
            'capability' => 'edit_pages',
            'position' => '50',
        ]);

        // onOffice WP-Website > Design
        acf_add_options_page([
            'page_title' => __('Gestaltung', 'oo_theme'),
            'menu_title' => __('Gestaltung', 'oo_theme'),
            'parent_slug' => 'oo-website',
            'capability' => 'edit_pages',
            'menu_slug' => 'oo-design',
        ]);

        // onOffice WP-Website > Header
        acf_add_options_sub_page([
            'page_title' => __('Header', 'oo_theme'),
            'menu_title' => __('Header', 'oo_theme'),
            'parent_slug' => 'oo-website',
            'capability' => 'edit_pages',
            'menu_slug' => 'oo-header',
        ]);
    }
}
add_action('acf/init', 'oo_acf_options_page');
```

**Comment by yeneastgate** [comment]
@fredericalpers I fixed this bug. Please take a look at my video demo. Thanks! 

https://github.com/onOffice-Web-Org/oo-wp-plugin/assets/104892108/d394efe7-d786-4201-96a1-2786f92d9df4

**Comment by fredericalpers** [comment]
@yeneastgate thank you for the quick fix, I will move this to v4.16 so we can release it asap, thank you :)
